test/test_gpu3.cpp

```diff
@@ -8099,6 +8099,69 @@ TEST_F(NVFuserTest, FusionManagedData_CUDA) {
   ASSERT_EQ(kernel->getManaged<T2>("data2").magic_number, 0x123456789abcdef);
 }
 
+TEST_F(NVFuserTest, FusionGeneratedTestCrossEntropyLoss_CUDA) {
+  std::unique_ptr<Fusion> fusion_ptr = std::make_unique<Fusion>();
+  auto fusion = fusion_ptr.get();
+  FusionGuard fg(fusion);
+
+  {
+    auto tv0 = TensorViewBuilder().ndims(2).contiguity({true, true}).dtype(DataType::Float).build();
+    fusion->addInput(tv0);
+    auto tv1 = TensorViewBuilder().ndims(1).contiguity({true}).dtype(DataType::Int).build();
+    fusion->addInput(tv1);
+    auto tv2 = max(tv0, {1});
+    auto tv3 = broadcast(tv2, {false, true});
+    auto tv4 = expand(tv3, 
+                      {IrBuilder::create<Int>(128),
+                       IrBuilder::create<Int>(371)});
+    auto tv5 = sub(tv0, tv4);
+    auto tv6 = exp(tv5);
+    auto tv7 = sum(tv6, {1});
+    auto tv8 = broadcast(tv7, {false, true});
+    auto tv9 = expand(tv8, 
+                      {IrBuilder::create<Int>(128),
+                       IrBuilder::create<Int>(371)});
+    auto tv10 = div(tv6, tv9);
+    auto tv11 = log(tv10);
+    auto tv12 = neg(tv11);
+    auto tv13 = reshape(tv1, {128}, {128, 1});
+    auto tv14 = torch_gather(tv12, 1, tv13);
+    auto s15 = IrBuilder::create<Int>(5);
+    auto tv16 = eq(tv13, s15);
+    auto s17 = IrBuilder::create<Double>(0.0);
+    auto tv18 = where(tv16, s17, tv14);
+    auto tv19 = sum(tv18, {0, 1});
+    auto tv20 = castOp(DataType::Float, tv16);
+    auto tv21 = sum(tv20, {0, 1});
+    auto s22 = IrBuilder::create<Double>(128.0);
+    auto tv23 = sub(s22, tv21);
+    auto tv24 = div(tv19, tv23);
+    fusion->addOutput(tv24);
+  }
+
+  auto options = at::TensorOptions().dtype(at::kFloat).device(at::kCUDA, 0);
+  std::vector<c10::IValue> inputs;
+  std::vector<at::Tensor> outputs;
+
+  {
+    auto t0 = at::randn({128, 371}, options);
+    inputs.push_back(t0);
+    auto t1 = at::randint(371, {128}, options).to(at::ScalarType::Long);
+    inputs.push_back(t1);
+    // note: reduction arg
+    //   none -> 0
+    //   mean -> 1
+    //   sum  -> 2
+    auto t24 = at::cross_entropy_loss_symint(t0, t1, None, 1, 5, 0.0);
```

Comment:
`None` is not valid in C++. What's the right way to call this function?

Reply:
Never mind, it's just `{}`

Reply:
Oops. sorry about this, must have copied the wrong file :bow: